guides/setup.md

```diff
@@ -0,0 +1,95 @@
+---
+title: DustJS | Setup
+layout: guides
+permalink: /guides/setup/
+---
+
+# Installing Dust
+
+## npm
+
+To install as a Node module:
+
+```
+npm install --save dustjs-linkedin
+```
+
+Dust should work with any version of Node or io.js newer than 0.8.12.
+
+The command-line compiler is located at `./node_modules/.bin/dustc`
+
+## Bower
+
+To install using Bower:
+
+```
+bower install dustjs-linkedin
+```
+
+Dust is compatible with IE7+ and all modern browsers.
+
+## Download
+
+The latest version of Dust is available on the GitHub <a target="_blank" href="https://github.com/linkedin/dustjs/tree/master/dist">page</a>
+(or <a target="_blank" href="">past versions</a>).
```

Comment:
No one cares about past versions and if you do you know enough to go find them yourself